instrumentation/netty/README.md

```diff
@@ -0,0 +1,35 @@
+# brave-instrumentation-netty
+
+This module contains netty http handler.
+The filters extract trace state from incoming requests. Then, they
+reports Zipkin how long each request takes, along with relevant tags
+like the http url. The exception handler ensures any errors are also
+sent to Zipkin.
+
+To enable tracing you need to do these:
+```java
```

Comment:
can you change this example to not use test classes?

Perhaps mention the basics which are that you need to add below to your pipeline:

```java
p.addLast("braveResponse", NettyTracing.create(httpTracing).createHttpResponseHandler());
p.addLast("braveRequest", NettyTracing.create(httpTracing).createHttpRequestHandler());
```